frontend/src/types/CamperTypes.ts

```diff
@@ -34,6 +34,8 @@ export type Camper = {
   ];
 };
 
+export type CreateCamperRequest = Omit<Camper, "id">;
```

Comment:
Could we use the `RegistrantExperienceCamper` type in dev instead? The multi-session backend PR made some changes to the data being passed in so this type would be more up to date (I pushed the commits to Thomas's step 1 PR  to make the types more consistent with my backend PR)